apps/web/src/app/globals.css

```
@@ -72,21 +72,33 @@ html {
   scroll-behavior: smooth;
 }
 
+/* page titles: /blog index h1 + individual post article title */
+.blog-heading {
+  @apply text-4xl lg:text-5xl font-bold tracking-tight leading-tight;
+  font-family: Helvetica, Arial, sans-serif;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)